src/html.js

```diff
@@ -17,16 +17,6 @@ export default function HTML(props) {
                 {props.preBodyComponents}
                 <div key={`body`} id="___gatsby" dangerouslySetInnerHTML={{ __html: props.body }} />
                 {props.postBodyComponents}
-                <script
```

Comment:
We don't need src/html.js anymore I think?

Reply:
ya, we dont need it anymore for now, lets remove it